docs/topics/installing-cli-tool.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// * docs/cli-guide/master.adoc
+
+:_content-type: PROCEDURE
+
+[id="installing-cli-tool_{context}"]
+= Installing the {CLINameTitle}
+
+You can install the {CLINameTitle} on Linux, Windows, or macOS operating systems.
+
+
+.Prerequisites
+
+* Podman must be installed
+
++
+[NOTE]
+.Podman
+====
+Podman is a daemonless, open source, Linux native tool designed to make it easy to find, run, build, share and deploy applications using Open Containers Initiative (OCI) Containers and Container Images.
+Podman provides a command line interface (CLI) familiar to anyone who has used the Docker Container Engine.
+For more information on installing and using Podman, see link:https://podman.io/docs/installation[Podman installation instructions].
+====
+
+[id="installing-downloadable-cli-zip_{context}"]
+== Installing the {CLINameTitle} `.zip` file
+
+.Procedure
+
+To install using the downloadable `.zip` file:
+
+. Navigate to the link:{DevDownloadPageURL}[{ProductShortName} Download page] and download the OS specific CLI file or the `src` file:
++
+* mta-7.0.0-cli-linux.zip
+* mta-7.0.0-cli-macos.zip
+* mta-7.0.0-cli-windows.zip
+* mta-7.0.0-cli-src.zip
+
+. Extract the `.zip` file to a directory of your choice. The `.zip` file extracts a single binary, called *mta-cli*.
+
++
+[NOTE]
+====
+If you are installing on a Windows operating system:
+
+. Extract the `.zip` file to a folder named `LC_PSN` to avoid a `Path too long` error. Alternatively, extract the file with link:https://www.7-zip.org/download.html[7-Zip] to a folder of any name you choose.
+. If a *Confirm file replace* window is displayed during extraction, click *Yes to all*.
+====
+
++
+When you encounter `<{ProductShortName}_HOME>` in this guide, replace it with the actual path to your {ProductShortName} installation.
+
+[id="installing-cli-zip_{context}"]
+== Installing the {CLINameTitle} using Podman
+
+.Procedure
+
+To install using `podman pull`:
+
+. Issue:
++
+[source,terminal]
+----
+podman cp $(podman create registry.redhat.com/mta-toolkit/mta-mta-cli-rhel9:{ProductVersion}):/usr/local/bin/mta-cli ./
```

Comment:
The file the user has to fetch from the container differs depending of the OS: 

* `mta-cli` for linux
* `darwin-mta-cli` for macOs
* `windows-mta-cli` for Windows.

At lest that was the case for the stage builds, I guess it'll be the same for GA. Is that right @dymurray?

Reply:
@dymurray @abrugaro 

```
podman cp $(podman create registry.redhat.io/mta/mta-cli-rhel9:7.0.0):/usr/local/bin/mta-cli ./

Trying to pull registry.redhat.io/mta/mta-cli-rhel9:7.0.0...
Error: initializing source docker://registry.redhat.io/mta/mta-cli-rhel9:7.0.0: unable to retrieve auth token: invalid username/password: unauthorized: Please login to the Red Hat Registry using your Customer Portal credentials. Further instructions can be found here: https://access.redhat.com/RegistryAuthentication
Error: name or ID cannot be empty
```

Should I comment out podman instructions and come back to it after release

Reply:
@anarnold97 requires a valid login to `registry.redhat.io`. so `podman login registry.redhat.io`